CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# HERMES development workflow
+
+## Preamble
+
+> **Branching is cheap!**
+>
+> Work in small packets, put up small pull requests!
+>
+> Aim for quick turnaround times!
+
+## Branching
+
+We loosely follow a mixture of [GitHubFlow](https://docs.github.com/en/get-started/quickstart/github-flow) and [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/):
+
+`main`
+    - Stable branch
+    - Merges come only from `develop` or a hotfix branch (i.e., when something needs to be fixed in "production")
+
+`develop`
+    - Unstable development branch
+    - Last "stable" is tagged: tag before attempting to break something
+
+`feature/<describe-feature>` (including issue id when exists: `feature/62-fix-broken-thing`)
+    - Branch from last tag on `develop`
+    - Feature branches may pick up "older" branches if they are not to complicated
+    - Combined branches must all start from the same tag
+
+### Pull requests
+
+- As soon as you have made 1 commit in a feature branch, put up a *draft* pull request
+- Keep pull requests small :skull: 
+- :warning: No pre-emptive reviews on PR drafts, **unless** the PR author @-mentions with this specific request
+- When you think you're done, mark PR ready for review, see below (Merge Process)
+
+## Merge Process (into `develop`)
+
+- Create PR from `feature/...` against `develop` (PR author)
+- Describe work in initial comment (PR author)
+    - Issue reference (`#n`)
```

Comment:
**suggestion**: it should use the closes/relates/... syntax

Reply:
"Relates" is not [syntax](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests), but we can invent one in the PR template. (I think a plain `- Related: #n, #m` is best.)
I personally like `Fix` in analogy to the commit message suggestions.

Reply:
True, but still it is kinda nice to read "Related to"/"Closes"/"Fixes" instead of just an issue number, not knowing what is the relation between PR and issue.

I'm fine with using "Fix"/"Fixes" - no more bike shedding :wink: